### PR TITLE
Enable sensor data to be sent to server

### PR DESCRIPTION
**Pull request by poornas**

### CODE REVIEW

File: app/src/main/java/com/minio/io/alice/SensorDataLogger.java [hunk 1]
```diff
@@ -58,7 +58,7 @@ public void onSensorChanged(SensorEvent event) {
             lastUpdate = System.currentTimeMillis();
             SensorRecord record = new SensorRecord(event);
             //Commented out until server can accept sensor data
-            //write(record.toString());
+            write(record.toString());
```
Comment (harshavardhana) [inline app/src/main/java/com/minio/io/alice/SensorDataLogger.java]: This is already done.. @poornas

Comment (harshavardhana) [inline app/src/main/java/com/minio/io/alice/SensorDataLogger.java]: Oh my bad i see this is sensor data not location

File: app/src/main/java/com/minio/io/alice/ClientWebSocket.java [hunk 2]
```diff
@@ -41,7 +41,7 @@
     public void connect(Context context) {
         this.context = context;
 
-        final String wsuri = "ws://192.168.1.225:8080";
+        final String wsuri = "ws://192.168.1.51:8080";
```
Comment (harshavardhana) [inline app/src/main/java/com/minio/io/alice/ClientWebSocket.java]: You don't need to change this, perhaps keep these changes locally only.

File: local.properties [hunk 3]
```diff
@@ -7,6 +7,5 @@
 # Location of the SDK. This is only used by Gradle.
 # For customization when using a Version Control System, please read the
 # header note.
-#Tue Feb 21 14:32:23 PST 2017
```
Comment (harshavardhana) [inline local.properties]: We need to figure out a way to avoid this for every commit @deekoder - i think this file should be removed from the repo.

Comment (deekoder) [inline local.properties]: we need the file in the repo. But this issue will go away once you host server.

File: app/src/main/java/com/minio/io/alice/ClientWebSocket.java (hunk 2 above)
Comment (deekoder) [inline app/src/main/java/com/minio/io/alice/ClientWebSocket.java]: @poornas can you amend this commit to exclude this file.

File: local.properties (hunk 3 above)
Comment (harshavardhana) [inline local.properties]: I think it should be part of android studio which maintains this file for us, keeping in repo keeps this file to creep in all the time.

**Comment by poornas** [comment]
Omitting ClientWebSocket.java and local.properties from tracking

**Pull request merged**